src/callback-endpoint.ts

```diff
@@ -165,8 +165,8 @@ export const createCallbackEndpoint = (
       // generate and set cookie
       // /////////////////////////////////////
       const cookie = generatePayloadCookie({
-        collectionConfig,
-        payload: req.payload,
+        collectionAuthConfig: collectionConfig.auth,
+        cookiePrefix: "payload",
```

Comment:
This can be pulled from existing payload config rather than hard coding.

Reply:
Ah, did not think of that. Fixed.